jest.config.js

```diff
@@ -1,11 +1,18 @@
-module.exports = {
+const nextJest = require('next/jest')
+
+const createJestConfig = nextJest({
+  // Provide the path to your Next.js app to load next.config.js and .env files in your test environment
+  dir: './',
+})
+
+module.exports = createJestConfig({
   roots: ['<rootDir>/src'],
   testMatch: ['**/__tests__/**/*.spec.[jt]s?(x)'],
   setupFiles: ['<rootDir>/.jest/storyBookTesting.ts'],
   setupFilesAfterEnv: ['<rootDir>/.jest/jestDomSetup.ts'],
-  testEnvironment: 'jsdom',
+  testEnvironment: 'jest-environment-jsdom',
   globalSetup: '<rootDir>/.jest/globalSetup.ts',
-  transform: {
-    '\\.(js|jsx|tsx|ts)$': ['babel-jest', { configFile: './jest.babelrc' }],
+  moduleNameMapper: {
+    '\\.(scss|sass|css)$': 'identity-obj-proxy',
```

Comment:
what does identity-obj-proxy do? i read it's doc but it wasn't clear. can you give an example?

Reply:
it's pretty much a mock. that returns same in same out.